SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -193,6 +194,7 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	FFilterPredicate ActorFilter;
 	TArray<FDispatcherRefreshCallback> ActorRefreshCallbacks;
 
+	TUniquePtr<FSpatialDebuggerSystem> SpatialDebuggerSystem;
```

Comment:
I'm in two minds on TUniquePtr usage for the GDK - why we use raw pointers rather than private `TSharedPtr` in the NetDriver with `TWeakPtr` Get() functions?